btr-api/tests/unit/resources/test_requests.py

```diff
@@ -231,7 +241,7 @@ def test_get_comment(app, client, session, jwt, requests_mock, sample_user, test
 
 @pytest.mark.parametrize(
     'test_name, user_type',
-    [('test_get_comment_no_auth', UserType.USER_COMPETENT_AUTHORITY)],
+    [('test_get_comment_no_auth', UserType.USER_PUBLIC)],
```

Comment:
same thing here we don't allow USER_COMPETENT_AUTHORITY either so should have both test cases?